notebooks/3_KernelMethods.ipynb

```diff
@@ -119,7 +142,7 @@
     "\\quad \\mathbf{K} = \\mathbf{\\Phi_X\\Phi_X^T}.\n",
     "\\end{equation}\n",
     "\n",
-    "$\\mathbf{\\Phi_X}$ indicates a matrix that holds the values of the RKHS features for each points in $\\mathbf{X}$. To simplify the notation, in this notebook we drop the $\\mathbf{X}$ subscript and refer to this matrix simply as $\\mathbf{\\Phi}$."
+    "$\\mathbf{\\Phi_X}$ indicates a matrix that holds the values of the RKHS features for each sample in $\\mathbf{X}$. The size of $\\mathbf{X}$ and $\\mathbf{\\Phi_X}$  are $n_{samples}\\times n_{original\\ features}$ and $n_{samples}\\times n_{RKHS\\ features}$ To simplify the notation, in this notebook we drop the $\\mathbf{X}$ subscript and refer to this matrix simply as $\\mathbf{\\Phi}$."
```

Comment:
Do not specify the size of the RKHS vectors, because it is potentially infinite, so the size is innumerable.

Reply:
Okay. I did this because we used the approximate RKHS feature matrix many times later. I thought it would be helpful.

![image](https://user-images.githubusercontent.com/56018055/97763533-2ccb2380-1b0c-11eb-9261-7a4669c975c3.png)

Reply:
Do I need to delete the part about the size of the RKHS vectors?

Reply:
Maybe the best way to do this is to not give the explicit sizes, but just emphasize that depending on the function phi, the number of features in X and Phi can be different